build.sbt

```diff
@@ -75,11 +75,11 @@ lazy val test             = Project("daffodil-test", file("daffodil-test")).conf
                               .settings(commonSettings, nopublish)
 
 lazy val testIBM1         = Project("daffodil-test-ibm1", file("daffodil-test-ibm1")).configs(TestDebug)
-                              .dependsOn(tdml)
+                              .dependsOn(tdml, core % "test->test")
                               .settings(commonSettings, nopublish)
 
 lazy val tutorials        = Project("daffodil-tutorials", file("tutorials")).configs(TestDebug)
-                              .dependsOn(tdml)
+                              .dependsOn(tdml, core % "test->test")
```

Comment:
These new dependencies might require modifications to the Eclipse project/classpath files? Mike, could use give this a quick test and let me know if changes are needed? Maybe even add a pull request to this branch on stevedlarwence/incubator-daffodil if so? I don't have an eclipse environment set up so that would take a while to test for me.

Reply:
I will fix these eclipse project classpath things. I would prefer to let you break them with this PR, then do a straight PR to fix them. Just to avoid confusion, as I have too many balls in the air at the moment.

Reply:
Ok, works for me.